CMake/AbseilHelpers.cmake

```diff
@@ -110,6 +116,7 @@ function(absl_cc_library)
       add_library(${_NAME} "")
       target_sources(${_NAME} PRIVATE ${ABSL_CC_LIB_SRCS} ${ABSL_CC_LIB_HDRS})
       target_include_directories(${_NAME}
+        ${ABSL_DECLARE_PUBLIC_SYSTEM_HEADERS_KEYWORD}
```

Comment:
Proposal: since `ABSL_DECLARE_PUBLIC_SYSTEM_HEADERS` is only use here would it be possible to use
use a generator expression e.g. (not tested !!!)
```cmake
$<${ABSL_DECLARE_PUBLIC_SYSTEM_HEADERS}:"SYSTEM">
```
and remove lines 83-88 above...